src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result -= 50;
+  } else if (days >= 3) {
+    result -= 20;
+  }
+
+  return result;
```

Comment:
The function name `calculateRentalCost` is good as it is descriptive of what the function does. However, the code contains hard-coded literals for the daily rate, long term rental days, and discounts, which should be replaced by constants to make the code more readable and maintainable. Refer to checklist item 2.